### PR TITLE
Display upcoming hackathons

### PR DESCRIPTION
**Pull request by Jays-T**
## Description

Now rendering dataset Hackathons in the Upcoming Hackathon section

## Updates

Updated the index.html view to render Hackathons from the database
Updated index.html to render each Hackathon.

landing_hackathon
landing_hackathon_1

### CODE REVIEW

File: home/templates/home/index.html [hunk 1]
```diff
@@ -5,50 +5,40 @@
...
-                    <div id="collapseThree" class="collapse" aria-labelledby="headingThree" data-parent="#accordion-upcoming-hackathons">
-                    <div class="card-body">
-                        Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf moon tempor, sunt aliqua put a bird on it squid single-origin coffee nulla assumenda shoreditch et. Nihil anim keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident. Ad vegan excepteur butcher vice lomo. Leggings occaecat craft beer farm-to-table, raw denim aesthetic synth nesciunt you probably haven't heard of them accusamus labore sustainable VHS.
-                    </div>
-                    </div>
-                </div>
+                {% if hackathons %}
+                {% for hackathon in hackathons %}
+                    {% if hackathon.organisation == 'Code Institute' or hackathon.organisation == user.organisation %}
+                        {% if hackathon.status == 'published' or user == hackathon.created_by %}
```
Comment (stefdworschak) [inline home/templates/home/index.html:13]: Could you just change this to only display `published` hackathons?

File: home/views.py [hunk 2]
```diff
@@ -1,10 +1,17 @@
 from django.shortcuts import render
 
+from hackathon.models import Hackathon
+
 
 def index(request):
     """ A view to return the index page """
+    hackathons = Hackathon.objects.all()
+
+    context = {
```
Comment (stefdworschak) [inline home/views.py:10]: I don't think you need to declare an extra `context` dict for only one key, just pass the dict directly into the `render` function.

**Review by stefdworschak — APPROVED** [review]
LGTM

@Jays-T just two small comments. Please revert back when you have added your changes and I will merge it.

File: home/views.py (hunk 2 above)
Comment (Jays-T) [reply home/views.py:10]: When doing that I get the error, "context must be a dict and not a queryset". I also figured that we might want to declare extra information for the landing page in future as the project grows.

File: home/templates/home/index.html (hunk 1 above)
Comment (Jays-T) [reply home/templates/home/index.html:13]: Done

**Pull request merged**